plugin-gradle/README.md

```diff
@@ -468,8 +468,28 @@ spotless {
 }
 ```
 
+<a name="prettier-plugins"></a>
+### Using community plugins for prettier
+
+Since spotless uses the actual npm prettier package behind the scenes, it is possible to use prettier with
+community-plugins in order to support even more file types.
+
+```gradle
+spotless {
+  format 'prettierJava', {
+    target '**/*.java'
```

Comment:
This is consistent with the rest of the README, but we should stop encouraging people to use `**/`.  Can we change this to `src/*/java/**/*.java`.  Something similar for php, at least `src/**/*.php`.

Reply:
Sure, do you think I should change that globally while I’m at it?

Reply:
changed the ones I found